docs/windows.md

```diff
@@ -246,6 +249,8 @@ kubectl apply -f kube-proxy.yml
 
 #### Add Windows antrea-agent DaemonSet
 
+Starting from Antrea 1.13, you need to manually set the kubeAPIServerOverride field in the YAML configuration file as the proxy-all mode is enabled by default.
+
```

Comment:
Give an example of antrea-agent-config snippet to show how to update the field?

Reply:
Do you mean give command to get the apiserver address here?

Reply:
No, I mean you should gve an example to show that "kubeAPIServerride" is configured.

Reply:
Updated.